src/webrtc/call.js

```diff
@@ -1350,14 +1350,17 @@ module.exports.createNewMatrixCall = function(client, roomId, options) {
             !webRtc.RtcPeerConnection || !webRtc.getUserMedia) {
         return null; // WebRTC is not supported.
     }
+
+    const optionsForceTURN = options ? options.forceTURN : false;
+
     const opts = {
         webRtc: webRtc,
         client: client,
         URL: w.URL,
         roomId: roomId,
         turnServers: client.getTurnServers(),
         // call level options
-        forceTURN: options ? options.forceTURN : false,
+        forceTURN: client._forceTURN || optionsForceTURN,
```

Comment:
This feels more than deprecation because the functionality has changed.

Reply:
Functionality from this point should be the same? ie. if you pass true here, it'll only use relay candidates

Reply:
Oh hang on.. client._forceTURN didn't exist before so it's fine. My bad!